rust/agama-server/src/network/model.rs

```diff
@@ -820,6 +820,206 @@ pub struct IpConfig {
     pub routes4: Vec<IpRoute>,
     #[serde(skip_serializing_if = "Vec::is_empty")]
     pub routes6: Vec<IpRoute>,
+    pub dhcp4_settings: Option<Dhcp4Settings>,
+    pub dhcp6_settings: Option<Dhcp6Settings>,
+    pub ip6_privacy: Option<i32>,
+}
+
+#[skip_serializing_none]
```

Comment:
Not checked deeply but I guess we should skip serializing it in general if it is the default...
that is, default **send_hostname** and unset **client_id** and **iaid**.

Reply:
I think for the model it isn't as important as for e.g. the settings. I was basing client_id etc. mostly on `MacAddress` which also isn't skipped if unset. But if you prefer it not being serialized, I'm not against it.